avs-finalizer/tests/operatorUtilities.ts

```diff
@@ -5,9 +5,21 @@ import registryCoordinator from "./abis/RegistryCoordinator.json";
 
 // @ts-ignore
 import finalizerTaskManager from "./abis/FinalizerTaskManager.json";
+// @ts-ignore
+import blsApkRegistry from "./abis/BLSApkRegistryStorage.json";
+// @ts-ignore
+import stakeRegistry from "./abis/StakeRegistryStorage.json";
+// @ts-ignore
+import indexRegistry from "./abis/IndexRegistryStorage.json";
+
 import {DockerUtils} from "./DockerUtils";
 export const registryCoordinatorAddress = '0x851356ae760d987E095750cCeb3bC6014560891C'
```

Comment:
in case the deployment changes, the addresses change, how can we make sure these are in sync ?
who's responsible, devs or qa ?
could we rather rely on the deployment json ?
since we already have some of the contract addresses hardcoded in configs, can we at least use a single contract reference ?
the contracts have references to each other, eg. we can query stake registry address from regcoord contract

Reply:
When the deployment json is updated? ie, we spin some changes that imply new deployment and therefore new contract addresses, at what stage this json is updated?
( I guess we are talking about the /output/chainId/deployment.json, right?  ) 

* the contracts have references to each other, eg. we can query stake registry address from regcoord contract
No, please! I can be a headache, jumping from one contract to another to get those addresses.